internal/graph/storagewrapper.go

```diff
@@ -0,0 +1,162 @@
+package graph
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/cespare/xxhash/v2"
```

Comment:
what was the driving characteristic of xxhash that drove the decision to use it over something in the Go stdlib?

Reply:
unsure why we decided on this back when it was introduced for [`CheckRequestCacheKey`](https://github.com/openfga/openfga/blob/a2df7810d7faf1a34b21e5abecb7472da54b28d3/internal/graph/cached_resolver.go#L195)

we could benchmark an alternative non-crypto hash for sure